spec/features/commissioner_runs_a_draft_spec.rb

```diff
@@ -98,40 +99,110 @@
-    #   league_id: league.id)
-    # player = FactoryGirl.create(:player)
-    # users = {}
-    # teams = {}
-    # names = %w(a b c d e f g)
-    # names.each do |name|
-    #   users[name] = FactoryGirl.create(:user)
-    #   teams[name] = Team.create(name: name, league_id: league.id, user_id: users[name].id)
-    # end
+    commissioner = FactoryGirl.create(:user)
+    league = League.create(
+      name: "name",
+      member_amount: "8",
+      commissioner_id: commissioner.id)
+    team = Team.create(
+      name: "teamname",
+      user_id: commissioner.id,
+      league_id: league.id)
+    player = FactoryGirl.create(:player)
+    users = {}
+    teams = {}
+    names = %w(a b c d e f g)
+    names.each do |name|
+      users[name] = FactoryGirl.create(:user)
+      teams[name] = Team.create(name: name, league_id: league.id, user_id: users[name].id)
+    end
+    initial_roster_count = team.rosters.count
 
-    # visit new_user_session_path
-    # fill_in "Email", with: commissioner.email
-    # fill_in "Password", with: commissioner.password
-    # click_button "Log in"
+    visit new_user_session_path
+    fill_in "Email", with: commissioner.email
+    fill_in "Password", with: commissioner.password
+    click_button "Log in"
 
-    # click_link league.name
-    # click_link("Start Draft", :match => :first)
-    # click_button("Submit")
-    # save_and_open_page
+    visit new_league_draft_path(league)
+    click_button("Submit")
+    fill_in "Price", with: "1"
+    select team.name , from: "pick[team_id][]"
+    all(".hidden-player-id").last.set player.id
+    click_button "Sold"
+    expect(team.rosters.count).to eq(initial_roster_count + 1)
   end
 
   scenario "but is unable to assign already-assigned players" do
+    commissioner = FactoryGirl.create(:user)
+    league = League.create(
+      name: "name",
+      member_amount: "8",
+      commissioner_id: commissioner.id)
+    team = Team.create(
+      name: "teamname",
+      user_id: commissioner.id,
+      league_id: league.id)
+    player = FactoryGirl.create(:player)
+    users = {}
+    teams = {}
+    names = %w(a b c d e f g)
+    names.each do |name|
+      users[name] = FactoryGirl.create(:user)
+      teams[name] = Team.create(name: name, league_id: league.id, user_id: users[name].id)
+    end
+    initial_roster_count = team.rosters.count
+
+    visit new_user_session_path
+    fill_in "Email", with: commissioner.email
+    fill_in "Password", with: commissioner.password
+    click_button "Log in"
+
+    visit new_league_draft_path(league)
+    click_button("Submit")
+    expect(page.find(".player-name")).to have_content(player.first_name)
+    fill_in "Price", with: "1"
+    select team.name , from: "pick[team_id][]"
+    all(".hidden-player-id").last.set player.id
+    click_button "Sold"
+    expect(team.rosters.count).to eq(initial_roster_count + 1)
+    expect{page.find(".player-name")}.to raise_error(Capybara::ElementNotFound)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.